README.md

```diff
@@ -63,6 +63,18 @@ you we created a parser that is local. You can open it by copying the path /
 browser then upload the claim.json file of that run.
 ![overview](docs/assets/images/htmlpage.png)
 
+## Upload a previous feedback to the html parser page
+
+Users are able to add their feedback about their test run then download their
+own copy of their feedback via the parser HTML page.  Users can then re-use the
+feedback they have already entered in future runs.  To do this, users can
+upload their `feedback.json` file to the parser HTML page.  Alternatively,
```

Comment:
I don't understand this part: 

> Alternatively, users can place their `feedback.json` file in the path 
>  #your-code-path/cnf-certification-test/script/ and it needs to be named `feedback.js`."

 
Why do they need to place the file into that "script" folder? Users shouldn't care about the content of the cnf-`certification-test/results/html` folder, it's just for our code to use it in order to create the tar.gz file. This is my suggestion:

"In order to see that feedback in the HTML viewer/parser, it first needs to be converted to a javascript file called "feedback.js". Users can do this conversion using the `tnf` tool:
  ```
  ./tnf generate feedbackjs -f path/to/feedback.json -o my/output/folder
  ```
Where "my/output/folder" is a directory that must exist before running the command so the file "feedback.js" can be generated inside. Please note that this new javascript file must also be moved/copied to the same folder where the HTML files (results.html or results-embed.html) have been uncompressed so the web can use it. See the ["Results artifacts zip file" ](https://test-network-function.github.io/cnf-certification-test/test-output/#results-artifacts-zip-file) section of the "Test Output" tab in the documentation for more details."